src/app/scheduling/scheduling.module.ts

```diff
@@ -28,6 +29,8 @@ import { DatepickerSample7Component } from "./datepicker/datepicker-sample-7/dat
 import { DatepickerSample8Component } from "./datepicker/datepicker-sample-8/datepicker-sample-8.component";
 import { AutocompletePipeStartsWith, DatepickerSample9Component, PipeWithoutTownFrom
     } from "./datepicker/datepicker-sample-9/datepicker-sample-9.component";
+// tslint:disable-next-line: max-line-length
```

Comment:
remove // tslint:disable-next-line: max-line-length and spread the line in two lines